src/index.js

```diff
@@ -54,11 +61,21 @@ const unbindEvents = function CozyBarUnbindEvents () {
 }
 
 const getDefaultStackURL = function GetDefaultCozyURL () {
-  return document.querySelector('[role=application]').dataset.cozyDomain
+  const appNode = document.querySelector(APP_SELECTOR)
+  if (!appNode) {
+    console.warn(`Cozy-bar is looking for a "${APP_SELECTOR}" tag that contains your application and can't find it :'(… The BAR is now disabled`)
```

Comment:
Great, can we adapt the warning message to reflect the status? In this context, the bar will try to be injected, even if it probably fails at init if the cozyURL isn't supplied explicitely. I suggest something like: "Cozy-bar can't discover the cozy's URL, and will probably fail to initialize the connection with the stack"